### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by sophia-in-shao**

### CODE REVIEW

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 103
- Score: 95.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 9
- Score: 75.00%
---

**Comment by sophia-in-shao** [comment]
@iphysresearch

**Comment by iphysresearch** [comment]
> @iphysresearch

还差几个题都做对，再努力一下？ @sophia-in-shao

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 103
- Score: 95.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

File: 2023/homework/PingShen/homework-matplotlib_seaborn.ipynb
Comment (iphysresearch) [inline 2023/homework/PingShen/homework-matplotlib_seaborn.ipynb:1]: 完成Matplotlib数据可视化作业！

**Comment by iphysresearch** [comment]
@sophia-in-shao 要不把Python扩展作业的那五道题也做一些呗？

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @sophia-in-shao
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

File: 2023/homework/PingShen/homework-leetcode.ipynb
Comment (iphysresearch) [inline 2023/homework/PingShen/homework-leetcode.ipynb:1]: 完成Leetcode扩展作业！

File: 2023/homework/PingShen/homework-seaborn.ipynb
Comment (iphysresearch) [inline 2023/homework/PingShen/homework-seaborn.ipynb:1]: 完成seaborn数据可视化作业！

**Review by iphysresearch — APPROVED** [review]
@sophia-in-shao 
恭喜完成Python编程的基础和扩展作业+数据可视化扩展作业！

**Pull request merged**